topics/admin/tutorials/ansible/tutorial.md

```diff
@@ -102,9 +103,11 @@ We can look at the [ansible-cvmfs role](https://github.com/galaxyproject/ansible
 │   ├── main.yml
 │   └── ...
 ├── templates
-│   └── stratum1_squid.conf.j2
+│   ├── stratum1_squid.conf.j2
+│   └── ...
 └── vars
     ├── debian.yml
+    ├── main.yml
     └── redhat.yml
```

Comment:
```suggestion
    ├── ...
    ├── main.yml
    └── ...
```

Reply:
I suggest keeping the distribution names here: they are mentioned as an example in the text describing the `var` folder.

Reply:
The 2 files have been renamed in the role. I don't think we need to mention them explicitly any way, since the `vars` description below doesn't rely on having separate files to define the variables.